state-chain/runtime/src/lib.rs

```diff
@@ -1408,6 +1408,25 @@ impl_runtime_apis! {
 			all_prewitnessed_swaps
 		}
 
+		fn cf_scheduled_swaps(from: Asset, to: Asset) -> Vec<(Swap, BlockNumber)> {
```

Comment:
given we don't actually care about direction here, perhaps naming these just `asset1` and `asset2` would make this a bit clearer

Reply:
ok yeah, let's be consistent with the other ones, as per Alastair's comment